cpp/src/Glacier2/Blobject.cpp

```diff
@@ -3,8 +3,10 @@
 //
 
 #include "Blobject.h"
+
```

Comment:
Extra whitespace

Reply:
This fine, added by clang-format I believe. The first header that matches the name of the .cpp file is special and not included in the alphabetical sort.

Reply:
It was actually added by clang-tidy and I removed it.